src/Sarif.Converters/FxCopConverter.cs

```diff
@@ -422,14 +477,26 @@ public void RefineMessage(string checkId, string typename, string messageId, str
             public void RefineIssue(string message, string result, string certainty, string level, string path, string file, int? line)
```

Comment:
>RefineIssue [](start = 24, length = 11)

Also, does "Issue" here refer to a SARIF `Result` object? If you renamed the parameter `result` to `resolutionName`, would it be ok to rename this method to `RefineResult`?